src/twisted/internet/defer.py

```diff
@@ -1222,12 +1262,12 @@ def __init__(
             logged.  This does not prevent C{fireOnOneErrback} from working.
         """
         self._deferredList = list(deferredList)
-        self.resultList: List[Optional[_DeferredListResult]] = [None] * len(
+        self.resultList: List[Optional[Tuple[bool, _DeferredResultT]]] = [None] * len(
```

Comment:
A comment here explaining why `bool` is in here would be helpful, I think.

Reply:
I'll add a docstring for `resultList`.